etelemetry/tests/test_client.py

```diff
@@ -4,6 +4,20 @@
 from ..client import _etrequest, get_project, check_available_version
 
 
+def check_cxn():
```

Comment:
```suggestion
def check_cxn(scope="session"):
```
to not pester per each test and assume that tests are fast enough to not come through difference in network condition